src/components/App/App.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Header from '../Header/Header';
+import PrivateRoute from '../Utils/PrivateRoute';
+import PublicRoute from '../Utils/PublicRoute';
+import AuthContext from '../../context/AuthContext';
+import FavoritesPage from '../../routes/FavoritesPage/FavoritesPage';
+import HomePage from '../../routes/HomePage/HomePage';
+import LoginPage from '../../routes/LoginPage/LoginPage';
+import NotFoundPage from '../../routes/NotFoundPage';
+import AuthService from '../../services/auth-service';
+
+class App extends Component {
+  static contextType = AuthContext;
+
+  componentDidMount() {
+    this.context.setLoggedInState(AuthService.isLoggedIn());
+    this.context.setShelterAdminState(AuthService.isShelterAdmin());
+  }
+
+  render() {
+    return (
+      <div className='App'>
+        <header>
+          <Header />
+        </header>
+        <main>
+          <Routes>
+            <Route path={'/'} element={<HomePage />} />
+            <Route
+              path="/login"
```

Comment:
Not sure if I am doing something wrong, but I was unable to access this page

Reply:
It's because in the AuthService I have this temporary method implementation:
```
isLoggedIn() {
    // TODO
    return true;
  },
```
Since you're logged in and the login page is using PublicRoute, the app will redirect you to the home page.
On the other hand, if you change the isLoggedIn() method to return false, you can access the login page, but if you click on the favorites page you'll be redirected to the login page.

Reply:
Makes sense to me. Sounds like that will work great. Thanks for the clarification!